examples/source/e-commerce/Shopping_Cart.html

```diff
@@ -0,0 +1,121 @@
+<!---
+
+disablePlayground: true
```

Comment:
re-enable the playground

Reply:
It has been disabled on abe.com, but don't know why...

Reply:
I don't know either - I think it's fine to re-enable